css/style.css

```diff
@@ -395,19 +381,70 @@ h1,h2,h4 {
     flex-direction: column;
 }
 
-.content__reasons
-{   font-size: 40px;
+.content__title {
+    font-size: 40px;
     font-weight: 700;
     line-height: 55px;
-    
+    text-transform: uppercase;
+    margin: 145px 0 10px 0;
 }
 
-p {
+.content__text {
     font-size: 14px;
+    font-style: normal;
     font-weight: 500;
-    line-height: 20px; 
+    line-height: 20px;
     letter-spacing: 0.2px;
 }
 
+.grid-container {
+    display: grid;
+    align-content: space-around;
+    grid-template-columns: auto auto auto;
+    gap: 30px;
+    background-color: none;
```

Comment:
I don't think you need to specify background: none as there is no default background actually.

Reply:
+